docs/docs-content/release-notes.md

```diff
@@ -11,11 +11,47 @@ sidebar_custom_props:
 tags: ["release-notes"]
 ---
 
+## October 10, 2023 - Release 4.0.17
```

Comment:
I think this is supposed to be 4.0.18

Reply:
Thanks for catching a copy/paste error. Vel changed it to 4.0.19 in the ticket.